.gitignore

```diff
@@ -48,3 +48,4 @@ package-lock.json
 
 css/styles.css
 
+.cache
```

Comment:
what creates this dir? (also it'd be good to ensure all files have a trailing newline)

Reply:
this is a good change if it’s generated by a tool in the repo, but otherwise it should probably be in your global gitignore instead of in the repo

Reply:
@TaeKimJR Do you know what generated this / why it was added? I'm guessing it was part of the repo since @majapw has seen it as well, but it'd be good to have it documented at least in a discussion somewhere. I can add a comment as well if we figure it out :)

Reply:
I'm removing this. If it comes back, let's figure out why and add back w/ a comment! @TaeKimJR said it may have been storybook, but I can't reproduce it. :)